docs/pyalgotrad/index.md

```diff
@@ -11,76 +10,56 @@
         - [Upload your Strategy](upload.md)
         - [Test your Strategy](testing.md)
 
-### Visit the API
-
-- Packages
-    - [algobulls](package_algobulls.md)/
-    - [broker](package_broker.md)/
-    - [instrument](package_instrument.md)/
-    - [order](package_order.md)/
-    - [strategy](package_strategy.md)/
-    - [utils](package_utils.md)/
-- Modules:
-    - [constants](module_constants.md)
-
 # pyalgotrading
-
 Official Python Package for Algorithmic Trading APIs powered by AlgoBulls!
 
 ### Features
-
 - Powered by the [AlgoBulls Platform](https://algobulls.com)
 - Everything related to Algorithmic Trading Strategies!
     - Create & upload strategies on the [AlgoBulls Platform](https://algobulls.com)
     - Free pool of Strategies are available separately at [pyalgostrategypool](https://github.com/algobulls/pyalgostrategypool)!
     - Support for all 150+ Technical Indicators provided by [TA-Lib](https://mrjbq7.github.io/ta-lib/index.html)
     - Support for multiple candlesticks patterns - Japanese OHLC, Renko, Heikin-Ashi, Linebreak
-    - Support for multiple candle inntervals - 1 minute, 3 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 1 day.
+    - Support for multiple candle intervals - 1 minute, 3 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 1 day. 
     - Support for **Regular Orders** and **Bracket Orders**
     - Support for **MARKET**, **LIMIT**, **STOPLOSS**, **STOPLOSS-LIMIT** orders
     - Support for **INTRADAY** and **DELIVERY** orders
 - Support for **Backtesting**
 - Support for **Paper Trading**
 - Support for **Real Trading**
-- Support for multiple brokers for Real Trading
-    - Currently supported brokers - Zerodha
+- Support for multiple brokers for Real Trading 
+    - Currently supported brokers - Zerodha 
     - Coming soon - AliceBlue, Mastertrust
     - More on the way!
 - Real-time Logs for Backtesting, Paper Trading, Real Trading
-- Multiple real-time Reports available for Backtesting, Paper Trading and Real Trading -
+- Multiple real-time Reports available for Backtesting, Paper Trading and Real Trading - 
     - Profit-n-Loss report (PnL report)
     - Statistics of (PnL report)
     - Order History for each order with state transitions & timestamps
 - Plot Candlestick charts using [plotly.py](https://github.com/plotly/plotly.py)
-
+    
 Backtesting, Paper Trading and Real Trading can be performed on the same strategy code base!
 
 ### Documentation
-
 You can find the docs [here](https://algobulls.github.io/pyalgotrading/).
```

Comment:
do we need a section where the docs are pointing to themselves?

Reply:
done